### DISCUSSION TITLE
How about httpS and redirect (ports 80-443)?

### DISCUSSION THREAD

**Discussion by wawanUnic** [wg-easy/wg-easy#1021] [Q&A]
Hello.
Tell me, is there a solution to setting up this server for the protocol httpS ?
And how to configure a redirect from port 80 to port 443?
Thank you.

**Comment by eandersons** [comment]
Use a reverse proxy server (see for example #987 (which also points to #830)).
Search for "Nginx", "Caddy" or "Traefik" in [Wiki](/wg-easy/wg-easy/wiki), some of pages also contains instructions for SSL certificates.

> And how to configure a redirect from port 80 to port 443?

Depends on the chosen reverse proxy.
